platform/kube/controller.go

```diff
@@ -568,6 +569,34 @@ func (c *Controller) HostInstances(addrs map[string]bool) []*model.ServiceInstan
 	return out
 }
 
+// TODO error indicator
+func (c *Controller) GetIstioServiceAccounts(hostname string) []string {
```

Comment:
It's called when the envoy watcher is reloaded:
https://github.com/istio/manager/blob/4de0a10e2b87d776377022abee8e335d8783dfc4/proxy/envoy/watcher.go#L107
I will sync with Kuat about this.